### PR TITLE
Rotating Status

### PR DESCRIPTION
**Pull request by NoahDevelops**
Added a `Rotating Status` that changes every 8 seconds for people that want it!

### CODE REVIEW

File: bot.js [hunk 1]
```diff
@@ -284,6 +284,43 @@ Discord.on('message', msg => {
...
+//You HAVE TO HAVE 10 STATUSES FOR THIS TO WORK	AND THE LAYOUT MUST REMAIN THE SAME
+	setInterval(
+		function() 
+		{
+           let status = 
+	       statuses[Math.floor(Math.random() * statuses.length)];
+           Discord
+		   .user
+		   	.setActivity(status, {type: "WATCHING"});
+       }, 8);
```
Comment (velddev) [inline bot.js:320]: this is not 8 seconds, it's 8 milliseconds :^)

Comment (NoahDevelops) [reply bot.js:320]: Untrue

Comment (Misly16) [reply bot.js:320]: its 8 milliseconds

Comment (Misly16) [reply bot.js:320]: 8000 would be 8 seconds

Comment (NoahDevelops) [reply bot.js:320]: Ah shi let me add a package

Comment (ghost) [reply bot.js:320]: why would you need a package just to make it 8ms to 8000 :yikes:

Comment (boehs) [reply bot.js:320]: > this is not 8 seconds, it's 8 milliseconds :^)

I see no issue

**Review by ghost — COMMENTED** [review]
```js
//This allows you to make a custom status that rotates!
	let statuses = [
    'Status 1'
		,
  'Status 2'
		,
      'Status 3'
		,
  'Status 4'
		,
'Status 5
		',
     'Status 6'
		,
	              'Status 7'
		,
	        'Status 8'
		,
	                                         'Status 9'
		,
	            'Status 10'
		,
       ];
```

Just wondering, why is the array stretched across multiple lines?

```js
//You HAVE TO HAVE 10 STATUSES FOR THIS TO WORK	AND THE LAYOUT MUST REMAIN THE SAME
```

I don't think so?

**Comment by boehs** [comment]
> ```js
> //This allows you to make a custom status that rotates!
> 	let statuses = [
>     'Status 1'
> 		,
>   'Status 2'
> 		,
>       'Status 3'
> 		,
>   'Status 4'
> 		,
> 'Status 5
> 		',
>      'Status 6'
> 		,
> 	              'Status 7'
> 		,
> 	        'Status 8'
> 		,
> 	                                         'Status 9'
> 		,
> 	            'Status 10'
> 		,
>        ];
> ```
> 
> Just wondering, why is the array stretched across multiple lines?
> 
> ```js
> //You HAVE TO HAVE 10 STATUSES FOR THIS TO WORK	AND THE LAYOUT MUST REMAIN THE SAME
> ```
> 
> I don't think so?

**Pull request closed without merging**